docs/analysis-techniques.md

```diff
@@ -9,31 +9,31 @@
     - [5.1 Checking if the requirement can hold](#51-checking-if-the-requirement-can-hold)
     - [5.2 Checking if the requirement will always hold](#52-checking-if-the-requirement-will-always-hold)
   - [6 Symbolic execution](#6-symbolic-execution)
-    - [6.1 Introduction by example](#61-introduction-by-example)
+    - [6.1 Introduction by Example](#61-introduction-by-example)
     - [6.2 Limitations](#62-limitations)
   - [7 Static analysis](#7-static-analysis)
   - [8 A note of formal verification](#8-a-note-of-formal-verification)
   - [9 How Byont uses these techniques](#9-how-byont-uses-these-techniques)
-  - [10 Follow up](#10-follow-up)
+  - [10 Follow-up](#10-follow-up)
   - [More sources](#more-sources)
 
 ## 1 Unit testing
 
-Every developer writes unit-tests for there code (right?). The goal of such a unit test is to test that a single function does what it is expected to do. Or, depending on the usage, testing that a certain function calls other function to perform a local integration test. Sounds easy enough.
+Every developer writes unit tests for their code (right?). The goal of such a unit test is to test that a single function does what it is expected to do. Or, depending on the usage, testing that a certain function calls another function to perform a local integration test. Sounds easy enough.
 
 Let's have a simple function as seen below:
 
 ```solidity
 mapping(address => int256) credit;
 
-/// @notice Paying off existing dept, or adding credit.
+/// @notice Paying off existing dept or adding credit.
 /// @param amount The amount to add to the credit.
 function payOff(uint256 amount) public {
     credit[msg.sender] += int256(amount);
 }
 ```
 
-On first sight it seems just fine. However, not that an `uint256` is casted to an `int256`. Solidity uses [two's complement](https://nl.wikipedia.org/wiki/Two%27s_complement) for representing negative values. Meaning that where `uint256` has the range [0, $2^{256} - 1$], `int256` has the range [$-2^{255}$, $-2^{255} - 1$].
+At first sight, it seems just fine. However, note that an `uint256` is cast to an `int256`. Solidity uses [two's complement](https://nl.wikipedia.org/wiki/Two%27s_complement) for representing negative values. Meaning that where `uint256` has the range [0, $2^{256} - 1$], `int256` has the range [$-2^{255}$, $-2^{255} - 1$].
```

Comment:
a uint (long u is treated as a consonant)

Reply:
[$-2^{255}$, $-2^{255} - 1$]
$-2^{255}$ is not getting formatted for some reason

Reply:
Currently just using plain text to avoid rendering errors in other viewers: [afac119](https://github.com/Byont-Ventures/smart-contract-analysis-tools/pull/47/commits/afac1198df618d42b4f78a0b408d8372ecdcde8a)